phoenix-core/src/it/java/org/apache/phoenix/end2end/DerivedTableIT.java

```diff
@@ -313,21 +290,25 @@ public void testDerivedTableWithGroupBy() throws Exception {
             query = "SELECT DISTINCT COLLECTDISTINCT(t.b) FROM (SELECT b_string b, a_string a FROM aTable GROUP BY a_string, b_string) AS t GROUP BY t.a";
```

Comment:
@JamesRTaylor Do we have a deterministic ordering for arrays? If we do, I don't think this change should be made without finding out the cause.